### PR TITLE
chore(TradingReward): Adjust button align center on mobile

### PR DESCRIPTION
**Pull request by Woody810**
### Summary
📱🖥️🎨

Improved the responsiveness of the trading reward banner component on the web app. Used `useMatchBreakpoints` hook and aligned `Start Trading` button in `Banner.tsx`.

> _Responsive banner_
> _`useMatchBreakpoints` hook_
> _Aligns button well_

### Walkthrough
* Import and use `useMatchBreakpoints` hook to detect screen size and adjust layout ([link](https://github.com/pancakeswap/pancake-frontend/pull/7022/files?diff=unified&w=0#diff-03134056bc43863a3b5bcd8fe4cfeb2bf9f191923365ae993ce3259ea819a093L1-R1), [link](https://github.com/pancakeswap/pancake-frontend/pull/7022/files?diff=unified&w=0#diff-03134056bc43863a3b5bcd8fe4cfeb2bf9f191923365ae993ce3259ea819a093R117), [link](https://github.com/pancakeswap/pancake-frontend/pull/7022/files?diff=unified&w=0#diff-03134056bc43863a3b5bcd8fe4cfeb2bf9f191923365ae993ce3259ea819a093L161-R162))

Before:
image

After:
image

### CODE REVIEW

File: apps/web/src/views/TradingReward/components/Banner.tsx [hunk 1]
```diff
@@ -158,7 +159,7 @@ const TradingRewardBanner = () => {
           <Text bold mb="32px" maxWidth="404px" lineHeight="26.4px" fontSize={['16px', '16px', '16px', '24px']}>
             {t('Earn CAKE while trading your favorite tokens on PancakeSwap.')}
           </Text>
-          <Flex>
+          <Flex alignSelf={isMobile ? 'center' : 'auto'}>
```
Comment (ChefMomota) [inline apps/web/src/views/TradingReward/components/Banner.tsx:162]: alignSelf={['center', 'center', 'center', 'auto']}

Comment (Woody810) [reply apps/web/src/views/TradingReward/components/Banner.tsx]: fixed

**Comment by ChefMomota** [comment]
fix eslint

**Comment by Woody810** [comment]
> fix eslint

fixed🙏

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**